hathor/manager.py

```diff
@@ -89,7 +90,8 @@ def __init__(self, reactor: Reactor, *, pubsub: PubSubManager, peer_id: Optional
                  enable_sync_v1: bool = True, enable_sync_v2: bool = False,
                  capabilities: Optional[List[str]] = None, checkpoints: Optional[List[Checkpoint]] = None,
                  rng: Optional[Random] = None, soft_voided_tx_ids: Optional[Set[bytes]] = None,
-                 environment_info: Optional[EnvironmentInfo] = None) -> None:
+                 environment_info: Optional[EnvironmentInfo] = None,
+                 event_ws_factory: Optional[EventWebsocketFactory] = None) -> None:
```

Comment:
Why don't you let the builder prepare the `EventManger(...)` and receives it instead of the factory? I feel the factory is an internal part of the event manager and is not needed here.

Reply:
Agreed. Done! I had to refactor it a bit to be able to do this.

Reply:
Good job! Thanks!